src/styles/blocks/rating.scss

```diff
@@ -0,0 +1,36 @@
+.rating {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-bottom: 24px;
+
+  &__text {
+    font-family: $page-font;
```

Comment:
font weight 400 is default, and you use the same font everywhere, add it once to the html tag